Runtimes/Core/core/CMakeLists.txt

```diff
@@ -325,9 +325,12 @@ elseif(NOT APPLE AND NOT LINUX AND NOT ANDROID AND UNIX)
   target_link_libraries(swiftCore PRIVATE "${EXECINFO_LIBRARY}")
 endif()
 
-install(TARGETS swiftCore)
+install(TARGETS swiftCore
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}/swift$<$<NOT:$<BOOL:${BUILD_SHARED_LIBS}>>:_static>/${SwiftCore_PLATFORM_SUBDIR}/${SwiftCore_ARCH_SUBDIR}"
```

Comment:
I don't believe these are the right paths for either Darwin nor if we want to line up with Debian-based platforms properly since they like to put the libraries under the triple at the top-level, making the platform and arch subdirectories redundant. e.g: `/usr/lib/<triple>/swift/libswiftCore.so`.